mptt/admin.py

```diff
@@ -4,6 +4,8 @@
 from django.conf import settings
 from django.contrib.admin.actions import delete_selected
 from django.contrib.admin.options import ModelAdmin
+from django.contrib.admin.options import get_content_type_for_model
```

Comment:
Lines 6 and 7 import from the same module.

Reply:
I thought you wanted like this, since there were already 2 imports from 2 modules in separate lines :
`from django.contrib.admin.options import ModelAdmin` (line 6) and 
`from django.contrib.admin.options import IncorrectLookupParameters` (line 16) 
and
`from django.utils.encoding import force_text` (line 11) and 
`from django.utils.encoding import smart_text` (line 19).

Do you want me to put all 5 imports (lines 6, 7 and 16) in 2 different lines (since we have 3 imports from one module and 2 from another)?

Reply:
Ah right, I didn't see that; in my own projects I'm using black and isort so that I don't even have to think about formatting and sorting imports. 

I prefer one import per module, but you don't have to fix the code around it. I generally prefer one change per pull request but given that django-mptt does not see many changes these days the chance of conflicts is really small, so feel free to fix the surrounding code as well :)

Reply:
Just pushed all the corrections. Let me know if something else is missing.